Cargo.toml

```diff
@@ -1,60 +1,23 @@
 [package]
 name = "ark-ldt"
-version = "0.1.0"
+version = "1.1.0"
 authors = ["arkworks contributors"]
-edition = "2018"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+edition = "2021"
 
 [dependencies]
-ark-ff = { version = "^0.3.0", default-features = false }
-ark-std = { version = "^0.3.0", default-features = false }
-ark-r1cs-std = { version = "^0.3.0", default-features = false}
-ark-sponge = { version = "^0.3.0", default-features = false }
-ark-poly = { version = "0.3.0", default-features = false }
-ark-relations = { version = "^0.3.0", default-features = false, optional = true}
-tracing = { version = "0.1", default-features = false, features = [ "attributes" ], optional = true}
-
-[dev-dependencies]
-ark-test-curves = { version = "^0.3.0", default-features = false, features = ["bls12_381_scalar_field", "mnt4_753_scalar_field"] }
+ark-crypto-primitives = { version = "^0.4.0", default-features = false, features = ["merkle_tree", "sponge"] }
+ark-ff = { version = "0.4", default-features = false }
+ark-poly = { version = "0.4.2", default-features = false }
+ark-std = { version = "^0.4.0", default-features = false, features = ["std"] }
+ark-serialize = { version = "0.4", default-features = false }
+ark-test-curves = { version = "0.4", default-features = false, features = ["bls12_381_curve"] }
+itertools = { version = "0.13.0", default-features = false }
+lazy_static = { version = "1.4", default-features = false }
+poseidon-paramgen = { version = "0.4", default-features = false, features = ["std"]}
```

Comment:
@WizardOfMenlo @Pratyush `cargo build --target aarch64-unknown-none` seems nearly working, but not with poseidon-paramgen as a dependency. If you can think of some way to fix amazing, otherwise we choose one or the other for the time being?

Reply:
Feel free to kill poseidon-paramgen, I used it for testing but it should be removed for the time being (I also suspect I was misconfiguring it). If you need a PoseidonSponge for testing there should something in crypto-primitives to look at.

Reply:
Okay great. I nuked that.